en/simulation/README.md

```diff
@@ -129,6 +129,25 @@ make px4_sitl jmavsim
 <span></span>
 > **Note** To avoid PX4 detecting data link timeouts, increase the value of param [COM_DL_LOSS_T](../advanced/parameter_reference.md#COM_DL_LOSS_T) proportional to the simulation rate. For example, if `COM_DL_LOSS_T` is 10 in realtime, at 10x simulation rate increase to 100.
 
+### Lockstep Simulation
+
+SITL simulation is implemented using lockstep. This enables [running the simulation faster or slower than realtime](#simulation_speed), and also to pause it in order to step through code. This is achieved because the simulation (e.g. jMAVSim or Gazebo) and PX4 are running in lockstep meaning they wait on each other for their turn.
+The steps are:
+1. The simulation sends a sensor message [HIL_SENSOR](https://mavlink.io/en/messages/common.html#HIL_SENSOR) including a timestamp `time_usec` to update the sensor state and time of PX4.
+1. PX4 receives this and does one iteration of state estimation, controls, etc. and eventually send a actuator message [HIL_ACTUATOR_CONTROLS](https://mavlink.io/en/messages/common.html#HIL_ACTUATOR_CONTROLS).
+1. The simulation has waited until it receives the actuator/motor message, then simulates the physics and calculates the next sensor message to send to PX4 again.
+
+The system starts with a "freewheeling" period where the simulation sends sensor messages including time and therefore runs PX4 until it has initialized and responds with an actautor message.
+
+### Disable Lockstep Simulation
+
+If required the lockstep simulation can be disabled which means that the old behavior is used where both simulator and PX4 use the host system time.
```

Comment:
When/Why might this be required?

Reply:
If someone uses a proprietary simulator that doesn't support it.

Reply:
Thanks. Added a note